clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -65,11 +65,11 @@ void goo() {
   [[intelfpga::ivdep(0)]]
   for (int i = 0; i != 10; ++i)
     a[i] = 0;
-  // expected-warning@+1 {{'ii' attribute requires a positive integral compile time constant expression - attribute ignored}}
+  // expected-error@+1 {{'ii' attribute requires a positive integral compile time constant expression}}
```

Comment:
Why error?

Reply:
To align with ivdep attribute

Reply:
There were some good reasons, why I made these diagnostics to be warnings in the past, they these reasons might be incorrect in the end. Lets collect some inputs.

Reply:
Typically just ignoring attributes because they are incorrectly specified (like as when it comes to parameters to them) is user hostile, so the clang project typically does these as errors.  It isn't SUPER consistent about it, but that is typically the case.

Reply:
Okay, lets side with the errors then.